scripts/publish.sh

```diff
@@ -466,15 +509,18 @@ verify_ecr() {
 		docker pull ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:"$init"-${AWS_FOR_FLUENT_BIT_VERSION}
 		sha1_init=$(docker inspect --format='{{index .RepoDigests 0}}' ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:"$init"-${AWS_FOR_FLUENT_BIT_VERSION})
 	fi
-	docker pull ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:latest
-	sha2=$(docker inspect --format='{{index .RepoDigests 0}}' ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit:latest)
 
-	verify_sha $sha1 $sha2
+	if [ "${PUBLISH_LATEST}" = "true" ]; then
```

Comment:
The non publish latest version doesn't get verified right? And that's okay?

Reply:
ah good point... the logic should be improved here I think:

1. We should always verify the image version in the region against the one in public ECR.
2. If we are publishing a latest release, we should also check that the latest and version number images are the same.

Reply:
Number one is only for the sync task case. When in pipeline public ECR won't be published yet.

Reply:
I have updated the script to always check the version number tag in local region ECR against public ECR. If we are publishing latest, we also check against in region latest.

Reply:
This looks good!

Reply:
actually we can't verify against public ECR outside of sync task...